### PR TITLE
Shimmer loading

### PR DESCRIPTION
**Pull request by EricWVGG**

### CODE REVIEW

File: src/components/HomeParts/FeaturedArtPanel/index.tsx [hunk 1]
```diff
@@ -9,84 +12,119 @@ type IFeaturedArtPanel = {
...
-            <Icon glyph="palette" color="slate" darkColor="moon" level={2} outline glyphOutline label="Artifact 27" />
+            <Icon
+              loading={loading || !grant}
+              glyph="palette"
+              color="slate"
+              darkColor="moon"
+              level={2}
+              outline
+              glyphOutline
+              label="Artifact 27"
```
Comment (geometrizen) [inline src/components/HomeParts/FeaturedArtPanel/index.tsx:45]: artifact 27?

File: src/components/HomeParts/FeaturedArtPanel/index.tsx [hunk 2]
```diff
@@ -9,84 +12,119 @@ type IFeaturedArtPanel = {
...
+            <div>
+              <dt>Started</dt>
+              <dd>{formatDate(grant?.submission?.project?.creationDate)}</dd>
+            </div>
+            <div>
+              <dt>Completed</dt>
+              <dd>{formatDate(grant?.submission?.project?.completionDate)}</dd>
+            </div>
+          </List>
+
```
Comment (geometrizen) [inline src/components/HomeParts/FeaturedArtPanel/index.tsx:95]: you got the data you needed yeah?

File: src/copy/common/createProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const createProfile = {
+  headline: 'Complete your profile',
+  subtitle: 'This allows us to get you a final without delay and ensures that your donation is attributed.',
```
Comment (geometrizen) [inline src/copy/common/createProfile.tsx:3]: ugh do we need to ensure this doesn't overwrite the previous PR? :-/

**Review by geometrizen — COMMENTED** [review]
LGTM but check the few inline comments

File: src/components/HomeParts/FeaturedArtPanel/index.tsx (hunk 1 above)
Comment (EricWVGG) [reply src/components/HomeParts/FeaturedArtPanel/index.tsx:45]: waiting on stuff from Simon or Juju or whomever

Comment (EricWVGG) [reply src/components/HomeParts/FeaturedArtPanel/index.tsx:45]: added a todo note

File: src/components/HomeParts/FeaturedArtPanel/index.tsx [hunk 4]
```diff
@@ -9,84 +12,120 @@ type IFeaturedArtPanel = {
...
-          <dt>Started</dt>
-          <dd>{formatDate(grant.submission?.project?.creationDate)}</dd>
-        </div>
-        <div>
-          <dt>Completed</dt>
-          <dd>{formatDate(grant.submission?.project?.completionDate)}</dd>
-        </div>
-      </List>
-
-      {/* todo: waiting on @EK for data
```
Comment (geometrizen) [inline src/components/HomeParts/FeaturedArtPanel/index.tsx:60]: tell me more please

Comment (EricWVGG) [reply src/components/HomeParts/FeaturedArtPanel/index.tsx:60]: This is related to the "token #" stuff in the design. We can go over it later, when we're finalizing the artifact metadata stuff. I have to review everything, so the single todo isn't pertinent.

**Pull request merged**